.github/workflows/ci.yml

```diff
@@ -29,6 +29,8 @@ on:
         description: 'If true, this will skip the release job'
         type: boolean
         default: false
+      enable_phpmd:
+        type: boolean
```

Comment:
For safety I would add a `default: false` here as well, similar to the surrounding inputs.